packages/editor-ui/src/utils/htmlUtils.ts

```diff
@@ -5,8 +5,20 @@ import xss, { friendlyAttrValue } from 'xss';
 */
 
 export function sanitizeHtml(dirtyHtml: string) {
-	const allowedAttributes = ['href', 'name', 'target', 'title', 'class', 'id'];
-	const allowedTags = ['p', 'strong', 'b', 'code', 'a', 'br', 'i', 'em', 'small'];
+	const allowedAttributes = ['href', 'name', 'target', 'title', 'class', 'id', 'style'];
```

Comment:
can we move these out of the function body, so that these are reused and we aren't creating extra work for the GC.